cmd/cmd.go

```diff
@@ -1,93 +1,104 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"fmt"
 	"os"
-	"strings"
 	"time"
 
 	"github.com/Clever/batchcli/runner"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/sfn"
 )
 
 func main() {
-	functionCmd := flag.String("cmd", "", "The command to run")
-	localRun := flag.Bool("local", false, "Local mode - auto-assigns random AWS Batch config")
+	activityName := flag.String("name", "", "The activity name to register with AWS Step Functions")
+	activityCmd := flag.String("cmd", "", "The command to run to process activity tasks")
```

Comment:
we should just call this `cmd` I think given how many times we've changed the name.